tests/test_bilstm.py

```diff
@@ -0,0 +1,44 @@
+from wellcomeml.ml.bilstm import BiLSTMClassifier
+from wellcomeml.ml import KerasVectorizer
+from sklearn.pipeline import Pipeline
+import numpy as np
+
+
+def test_vanilla():
+    X = [
+        "One",
+        "One only",
+        "Two nothing else",
+        "Two and three"
+    ]
+    Y = np.array([0,0,1,1])
```

Comment:
I'm confused about the `Y` for the `multilabel =False` training. What's the rule that maps  `["One", "three", "one", "two", "four"]` to `[1,0,1,0,0]` and `["One","One only", "Two nothing else","Two and three"]` to `[0,0,1,1]`? or does it not matter since it's just a test.

Reply:
one -> 1
two -> 0

Reply:
I thought that was the rule, but doesn't that mean `["One","One only", "Two nothing else","Two and three"]` should be `[1,1,0,0]` then?

Reply:
sorry it is the other way around. does it matter?

Reply:
I guess not!